scripts/newinstall.sh

```diff
@@ -99,12 +117,11 @@ if [[ "$cont_flag" = false ]]; then
 	fi
 fi
 
-
-##########	Offer to get git if it's too old
+##########  Discuss the state of Git.
 
 if true; then
 	if hash git 2>/dev/null; then
-		GITVERNUM=$(git --version | cut -d" " -f 3)
+		GITVERNUM=$(git --version | cut -d\  -f 3)
```

Comment:
That's weird because that's exacty what I did - replaced the existing \d space space with -d" " - why is the diff the wrong way round?